index.html

```diff
@@ -23,15 +24,109 @@
+                    </div>
+            </div>
+
+            <div class="child">
+                <img src="./images/pexels-lucas-pezeta-3358299.jpg" width="55px" height="30px">
+                <div>
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment </div>      
+                    <div class="share">share</div>
+                    </div>
+            </div>
+      
+        
+            <div class="child">
+                    <img src="./images/pexels-mikky-koopac-625644.jpg" width="55px" height="30px">
+                    <div>
+                        <div class="material-icons">favorite_border</div>
+                        <div class="material-icons">mode_comment </div>      
+                        <div class="share">share</div>
+                    </div>
+            </div>
+        
+            <div class="child">
+                <img src="./images/pexels-nishant-das-3906333.jpg" width="55px"  height="30px">
+                    <div>
+                        <div class="material-icons">favorite_border</div>
+                        <div class="material-icons">mode_comment </div>      
+                        <div class="share">share</div>
+                    </div>
+            </div>
+
+            <div class="child">
+            <img src="./images/pexels-sebastian-ervi-1763067.jpg" width="55px" height="30px">
+                <div>
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment </div>      
+                    <div class="share">share</div>
+            </div>
+
+            </div>
+
+            <div class="child">
+                <img src="./images/pexels-sebastian-ervi-1763075.jpg" width="55px"  height="30px">
+                <div>  
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment </div>      
+                    <div class="share">share</div>
+                </div>
+            </div>
+
+            <div class="child">
+                <img src="./images/pexels-wendy-wei-1047940.jpg" width="55px" height="30px">
+                <div>
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment</div>      
+                    <div class="share">share</div>
+                </div>
+            </div>
+        
+            <div class="child">
+            <img src="./images/5655435.jpg" width="55px" height="30px">
+                <div>
+                    <div class="material-icons">favorite_border</div>
+                    <div class="material-icons">mode_comment</div>      
+                    <div class="share">share</div>
+                </div>
+            </div>
+        </div>
+
+<button type="button">prev</button>
+<button type="button">next</button>
+
+</article>
         </main>
```

Comment:
Formatting is incorrect.

Reply:
or should the button be inside the parent-container